tools/composer.json

```diff
@@ -1,9 +1,6 @@
 {
     "require": {
-        "friendsofphp/php-cs-fixer": "^3.16",
-        "phpstan/phpstan": "^1.10",
-        "phpunit/phpunit": "^9.6",
-        "phpspec/prophecy": "^1.11",
-        "phpspec/prophecy-phpunit": "^2.0"
+        "friendsofphp/php-cs-fixer": "3.16.0",
+        "phpstan/phpstan": "1.12.21"
```

Comment:
Why to not remove this file and move to root `dev` dependencies?

Reply:
That is a really good question. 

These are tools just like PHPStorm is a tool. I dont like them to be mixed up with normal dependencies and potentially block updates of "real" dependencies. 

Though, both phpstan and php-cs-fixer has solved this problem with phars which can be put in composer.json. 

I tend to just have them in the CI and not bother with putting them in local composer.json. But I did not want to change too much for the author and the community around this package. I just wanted to make the minimal changes possible to allow testing on 8.3.